measurements/perplexity/perplexity.py

```diff
@@ -100,7 +100,7 @@ def _info(self):
             reference_urls=["https://huggingface.co/docs/transformers/perplexity"],
         )
 
-    def _compute(self, data, model_id, batch_size: int = 16, add_start_token: bool = True, device=None):
```

Comment:
the convention is to use `data` as the primary column for measurements.

Reply:
Reverted, thanks!